routes/api/asset/index.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const router = express.Router();
+const Asset = require('../../../models/asset');
+
+// Search assets.
+router.get('/', (req, res) => {
+
+  let query = {};
+
+  if (typeof req.query.url !== 'undefined') {
+    query.url = req.query.url;
+  }
+
+  Asset.search(query)
+    .then((asset) => {
```

Comment:
It seems like `query` and `findByUrl` can both be used here, is there a reason why we to need to keep both functions?